src/theme/ThemeProvider.js

```diff
@@ -49,6 +58,7 @@ const hackcampTheme = {
     text: '#fff',
     link: '#fff',
     linkHover: '#FFBC96',
+    dropdown,
   },
   typography: {
     h1: {
```

Comment:
nwTheme and HackCamp theme have different colours, could you make two versions here? ty! 

for the HackCamp colours, you can consult Sandy Co (HackCamp design lead) on Slack, @jackyzha0 (HackCamp director), or @jvssicawu (livesite design lead)

Reply:
Resolved below in Ian's comment